src/android/TP6m.jsx

```diff
@@ -58,53 +113,35 @@ class TP6M extends React.Component {
 
     if (missing(referenceValue)) {
       // THERE IS NO GEOMEAN TO CALCULATE
-      this.setState({ summaryData: null, test, platform });
+      this.setState({
+        data: null,
+        referenceValue: null,
+        test,
+        platform,
+      });
 
       return;
     }
 
-    const summaryData = {
-      datasets: aggregate
-        .where({ test, platform })
-        .along('platform') // dummy (only one)
-        .map(({ result }) => ({
-          label: selectFrom(PLATFORMS)
-            .where({ platform })
-            .first().label,
-          data: result
-            .along('pushDate')
-            .map(point => ({
-              x: point.pushDate.getValue(),
-              y: point.getValue(),
-            }))
-            .toArray(),
-        }))
-        .append(
-          exists(referenceValue) && {
```

Comment:
notice the standard form deals with reference values easier, allowing us to remove this fake dataset that represents the reference value